fabric-rendering-api-v1/src/main/java/net/fabricmc/fabric/api/renderer/v1/RendererAccess.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.renderer.v1;
+
+import com.google.common.annotations.Beta;
+
+import net.fabricmc.fabric.impl.renderer.RendererAccessImpl;
+
+@Beta
```

Comment:
same as above

Reply:
Because it's bleeding edge stuff. Fix it if you can :)